scripts/setup_osx.sh

```diff
@@ -215,6 +215,9 @@ fi
 . "${CONDA_SH}"
 conda update -q -y conda
 
+# We need a MuJoCo key to import mujoco_py
+cp ${_arg_mjkey} "${HOME}"/.mujoco/mjkey.txt
```

Comment:
Same comment for the OSX setup

Reply:
Same reply but  in [here](https://github.com/rlworkgroup/garage/blob/master/scripts/setup_osx.sh#L135-L137)